### ISSUE TITLE
Disc textures display as errors on Aternos servers

### ISSUE THREAD

**Issue by Petrosmine** [TeamTernate/infinite-music-discs#76]
Datapack works perfectly, but the discs are just error blocks despite the resource pack being present in server and in client

**Comment by Petrosmine** [comment]
the previous version worked fine but after updating it to this version the textures broke

**Comment by bradytheinventor** [comment]
Related to #56? I tested the datapack a long time ago on Aternos but I'll take another look

**Comment by Petrosmine** [comment]
no for me the music plays its just the textures that completely break

**Comment by bradytheinventor** [comment]
Was your resourcepack from an older version? If so you should re-generate it with the latest version of the app. 1.19.4 changed resourcepacks a little (requires an atlas configuration file) so older resourcepacks will not display their custom textures correctly.

FYI: I generated a pack with 26 discs and uploaded it to an Aternos server and it worked. I used v1.5 of the app, with the "generate packs as zip" and "play tracks from the jukebox" settings enabled. I used the Aternos web UI to create a new server, added the datapack into my `world/datapacks` folder from the Files menu, and uploaded the resourcepack through the Options menu. After restarting the server, everything worked perfectly.

**Comment by bradytheinventor** [comment]
Not really, I can't predict how other resource packs are going to affect the game. You can give your custom music discs resourcepack the highest priority in the resourcepack menu to try and make sure the custom music discs override other resourcepacks. But there's no way to guarantee compatibility with every other resourcepack out there unfortunately
